packages/L10n/src/locales/es.yml

```diff
@@ -7,25 +7,32 @@ es:
     dataGrid:
       a11yTextMessage: "{{value}}. Está en la fila {{rowIndex}}, columna {{columnIndex}}. Haga caso omiso de la siguiente información:"
     uploader:
-      label: Arrastre y suelte o elija archivos de su computadora
-      overlay:
-        title: Arrastrar y soltar
-        subtitle: Sus archivos aquí
-      form:
-        dragAndDrop: Arrastrar y soltar
-        or: o
-        chooseFiles: Elija los archivos
-        from: De su computadora
+      drop_files: Soltar archivos a cargar
+      drop_files_here_or: Suelte los archivos a cargar aquí o
+      choose_from_computer: elija de su computadora
+      cancel_upload: Cancelar carga
+      restart_upload: Reiniciar carga
       errors:
-        file_size: excede el tamaño máximo del archivo de {{size}}.
-        file_type: "{{type}} es el tipo de archivo incorrecto."
+        size: El archivo debe tener menos de {{maxFileSize}}
+        type: "El archivo debe ser de uno de los siguientes tipos: {{- supportedTypes}}"
       status:
-        idle: ocioso
-        processing: en proceso...
-        uploading: cargando...
-        success: éxito
-        cancel: cancelar
-        error: error
+        idle: Ocioso
+        processing: En proceso
+        uploading: Cargando
+        success: Éxito
+        cancel: Cancelar
+        error: Error
+      progress:
+        cancelled: Cancelado/a
+        complete: Completado/a
+        idle: Ocioso
+        uploading: Carga de {{progressWithUnits}} de {{sizeWithUnits}}
```

Comment:
Source says: "Uploading {{progressWithUnits}} of {{sizeWithUnits}}"
I would use something different. For example "Cargando {{progressWithUnits}} de {{sizeWithUnits}}"
An example: "Cargando 1 de 5"
What do you think, @cristinajimenez?

Reply:
Totally agree.

Reply:
@agherreras and @cristinajimenez not sure if it is relevant, but in English it will say something like: `Uploading 143kb of 2.5mb` (not `Uploading 1 of 5`)

Reply:
Thanks @jamiek-acl for explaning. In that case I would use 
`Cargados {{progressWithUnits}} de {{sizeWithUnits}}`
Agree @agherreras ?

Reply:
Yes.
In this case we might also want to update other languages?
Portuguese also uses the gerund form of the verb.

Reply:
Maybe, but this is a stylistic choice, I don't think the gerund would be incorrect either. Could be fixed next batch if needed.

Reply:
Cool. Thanks for all the feedback! Will double check.

Reply:
Let's use Carregando (pt) and Cargando (es) for consistency with previous stylistic choice. Is it necessary to change it to "Cargados"?

Reply:
I like more "Cargados", since it indicates that amount of data has already being uploaded, which I think is accurate, but since source says "Uploading" instead of "Uploaded", I guess we can go with "Cargando"...

Reply:
I agree with Alberto, I would use "Cargados", it describes better what the action will do.

Reply:
Yes, let's not overdo it :-)

Reply:
"Uploading" is used in source rather than "Uploaded", since the action is in progress. Can we keep it simple and stick to the source, translating it as Carregando (pt) and Cargando (es)?

![image](https://user-images.githubusercontent.com/4226672/83914088-66e2f580-a725-11ea-9ff0-6e7efe6a5927.png)

Reply:
Sure